api/hydrophoneApi.go

```diff
@@ -446,3 +465,36 @@ func (a *Api) isAuthorizedUser(tokenData *shoreline.TokenData, userId string) bo
 		return false
 	}
 }
+
+// userId is admin of teamid
+// TODO to be updated with teams calls
+func (a *Api) isTeamAdmin(userId, teamId string) (bool, error) {
+	// testing purpose only
+	if userId == "76f47a4234" && teamId == "100" {
+		return false, nil
+	}
+	if userId == "c8d420ce52" && (teamId == "100" || teamId == "1000") {
+		return false, nil
+	}
+	return true, nil
+}
+
```

Comment:
Either create a new method in Crew_client: `GetMember(teamId, userId)` OR reuse the previous call to `GetTeam(teamId)` which should have already returned the list of members

Reply:
same as above